guides/source/active_support_core_extensions.md

```diff
@@ -164,7 +164,13 @@ Active Support provides `duplicable?` to programmatically query an object about
 false.duplicable? # => false
 ```
 
-By definition all objects are `duplicable?` except `nil`, `false`, `true`, symbols, numbers, class, module, and method objects.
+<<<<<<< HEAD
+By definition all objects are `duplicable?` excepting method objects, numbers as well as `nil`, `false`, `true`, symbols, class, and module.
+
+As of Ruby version 2.4.0 numbers, `nil`, `false`, `true`, symbols, class and module objects are also `duplicable?`.
+=======
+By definition all objects are `duplicable?` including BigDecimals, but excepting all other numbers as well as `nil`, `false`, `true`, symbols, class, module, and method objects. As of Ruby 2.4.0 even class and module objects are `duplicable?`.
+>>>>>>> d738fd8... Update list of `duplicable?` objects in AS core_ext guide [ci skip]
```

Comment:
This contains a merge conflict and the list post Ruby 2.4 goes a bit further: https://github.com/rails/rails/commit/2cb855812066b64e7e10e40096a4c4e8e8256afb 😊

Our minimum version is Ruby 2.2, so I don't think we need to mention that class and modules can't be dup'ed because I just tested them in my 2.2.5 console and they worked.

How about:

```markdown
In Ruby 2.4 most objects are `duplicable?` except method objects and certain numbers, whereas Ruby 2.2 and 2.3 can't duplicate `nil`, `false`, `true`, and symbols.
```

Can you dig in and find out which number types are left after BigDecimal and Numeric (which, I believe, unifies Fixnum and Bignum in 2.4)?

Note the avoidance of `excepting` as `except` is the correct version 😊